WordPress/UITests/Tests/EditorGutenbergTests.swift

```diff
@@ -53,6 +53,10 @@ class EditorGutenbergTests_01: EditorGutenbergTests {
 // The slowest one are in separate for better parallelization
 class EditorGutenbergTests_02: EditorGutenbergTests {
     func testBasicPostPublishWithCategoryAndTag() throws {
+        if #available(iOS 26.0, *) {
+            XCTExpectFailure("No photos in the Photos app")
+        }
+
```

Comment:
I have created an issue for these test failures: https://linear.app/a8c/issue/CMM-726